docker/package/defaults/tezos-baking.conf

```diff
@@ -0,0 +1,9 @@
+# vim: ft=sh
+# SPDX-FileCopyrightText: 2020 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+# shellcheck disable=SC2034
+DATA_DIR="/var/lib/tezos/.tezos-client"
+NODE_RPC_ENDPOINT="http://localhost:8732"
+BAKER_ADDRESS_ALIAS="baker"
```

Comment:
Two questions:

1. how come the `baker` alias is predefined here but not in `tezos-baker.conf` and `tezos-endorser.conf`?
    I'd imagine the same consideration apply for running single services and we can provide a default there too.
2. Is this file necessary? AFAIK the meta-service runs the two services, which in turn use the configs mentioned above, wouldn't it be sufficient to have those?

Reply:
> Is this file necessary?

Yes, it's used as an `EnvironmentFile` in `tezos-{baker, endorser}-<proto>@<network>` services
This config basically makes it possible to change `BAKER_ADDRESS_ALIAS` in one place and both baker and endorser services will use updated alias

> how come the baker alias is predefined here but not in tezos-baker.conf and tezos-endorser.conf?

Indeed, it probably makes sense to update them as well

Reply:
So, basically, now there are two services for each Tezos daemon:
1) Simple `tezos-{baker, endorser, accuser}-<proto>.service` that acts like previously and doesn't know anything about `tezos-baking-<network>.service`
2) Instantiated `tezos-{baker, endorser, accuser}-<proto>@.service` which is used as a part of `tezos-baking-<network>.service`, see [this](https://www.freedesktop.org/software/systemd/man/systemd.service.html#Service%20Templates)

Reply:
:ok_hand: I see, thanks.

> Indeed, it probably makes sense to update them as well

Ok, please do that, aside from that point the rest LGTM